src/main/java/org/apache/sysml/api/mlcontext/Script.java

```diff
@@ -243,14 +250,23 @@ public Script input(Map<String, Object> inputs) {
 	 *            Scala Map of inputs (parameters ($) and variables).
 	 * @return {@code this} Script object to allow chaining of methods
 	 */
-	public Script in(scala.collection.Map<String, Object> inputs) {
-		Map<String, Object> javaMap = JavaConversions.mapAsJavaMap(inputs);
+	@SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
Can you add a quick comment here when you merge indicating that we updated this to all Scala Maps to be passed in that would otherwise have the type `[String, Any]`, preventing it from being used from Java?